doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -7,6 +7,11 @@ This API changelog is experimental and we would love feedback on its usefulness.
     :local:
     :depth: 1
 
+v6.4
+----
+
+- ** /api/search?q=**: Json values for image_url in DataFiles and Collections have changed from Base64URL ("data:image/png;base64,...) to "/api/access/datafile/{identifier}?imageThumb=true" and "/api/access/dvCardImage/{identifier}" respectively. This was done to match the image_url of Dataset.
```

Comment:
Do we still need this release note if we merge this before we release 6.4?

Reply:
removed for now. can be re-added if this doesn't make 6.4